weaver/core/drivers/fabric-driver/tsconfig.json

```diff
@@ -15,7 +15,7 @@
     // "sourceMap": true,                     /* Generates corresponding '.map' file. */
     // "outFile": "./",                       /* Concatenate and emit output to single file. */
     "outDir": "./out",                        /* Redirect output structure to the directory. */
-    // "rootDir": "./",                       /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
+    // "rootDir": "./",                          /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
```

Comment:
Unnecessary whitespace?

Reply:
Actually was doing something with `rootDir` and reverted it, the whitespace got left, will remove it